examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -12,9 +12,8 @@
    "metadata": {},
```

Comment:
<p>Why is it not shift and flattened any more?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Cause it was a data augmentation method. This the inference pipeline without such methods. 
Also, now there is no `shift_flattened` function. This function shifted an image as well as the shift method of ImagesBatch. Now in the notebook used shift method of ImagesBatch (replace of shift_flattened for image shifting) and custom_filter (replace of shift_flattened as a user-defined function for data augmentation).